tools/permuter_settings.toml

```diff
@@ -9,3 +9,4 @@ OPEN_DISPS = "void"
 CLOSE_DISPS = "void"
 ABS = "int"
 SQ = "int"
+CLAMP = "int"
```

Comment:
commit intended?

Reply:
yes. I believe this macro should be added to the list